client/my-sites/site-settings/media-settings-performance.jsx

```diff
@@ -159,45 +149,13 @@ class MediaSettingsPerformance extends Component {
 	}
 
 	render() {
-		const {
-			isRequestingSettings,
-			isSavingSettings,
-			isVideoPressAvailable,
-			photonModuleUnavailable,
-			selectedSiteId,
-			siteId,
-			translate,
-			jetpackVersionSupportsLazyImages,
-		} = this.props;
-		const isRequestingOrSaving = isRequestingSettings || isSavingSettings;
+		const { isVideoPressAvailable, selectedSiteId } = this.props;
 
 		return (
 			<div className="site-settings__module-settings site-settings__media-settings">
-				{ ( ! jetpackVersionSupportsLazyImages || isVideoPressAvailable ) && (
+				{ isVideoPressAvailable && (
 					<Card>
 						<QueryJetpackConnection siteId={ selectedSiteId } />
```

Comment:
Can `<QueryJetpackConnection>` be removed? Were the `jetpack.connection` data used by anything else than the `isJetpackSiteInDevelopmentMode` selector?

Reply:
Nice find 👍 Cleaned up in https://github.com/Automattic/wp-calypso/pull/30040/commits/f55bf63f69ee20da1fd440258eb0206a94574ea2.

I'd appreciate another review when you get a chance.